Source: Same deal as http://old.nabble.com/nss_ldap-265-td26230441.html#a26236814
Upstream: In next release, I hope
Reason: Only crazy people have . in $PATH

--- pam_ldap-185.orig/Makefile.am	2009-11-06 20:29:34.000000000 +1000
+++ pam_ldap-185/Makefile.am	2010-10-04 22:23:06.301562203 +1000
@@ -44,4 +44,4 @@
 	@$(NORMAL_UNINSTALL)
 
 vers.c: $(top_srcdir)/CVSVersionInfo.txt
-	CVSVERSIONDIR=$(top_srcdir) vers_string -v
+	CVSVERSIONDIR=$(top_srcdir) $(top_srcdir)/vers_string -v
